keras/engine/topology.py

```diff
@@ -2923,6 +2923,31 @@ def preprocess_weights_for_loading(layer, weights,
                                                     (2, 3, 1, 0))
                 weights = [kernel, recurrent_kernel, bias]
 
+        if layer.__class__.__name__ in ['Model', 'Sequential']:
+            new_weights = []
+            # trainable weights
+            for sublayer in layer.layers:
+                nb_weights = len(sublayer.trainable_weights)
+                print('diving into', sublayer)
+                if nb_weights > 0:
+                    new_weights.extend(preprocess_weights_for_loading(layer=sublayer,
```

Comment:
If the following lines is aligned with the second parenthesis, it will cause [this error](https://travis-ci.org/fchollet/keras/jobs/243119322).